heron/config/src/yaml/conf/local/healthmgr.yaml

```diff
@@ -0,0 +1,10 @@
+## list of policies to be executed for self regulation
+#heron.class.health.policies:
+#  - dynamic-resource-allocation
+#
+## configuration specific to individual policies listed above
+#dynamic-resource-allocation:
+#  health.policy.class: "com.twitter.heron.healthmgr.policy.DynamicResourceAllocationPolicy"
+#  health.policy.interval.ms: "120000"
```

Comment:
I think a consistent value format, String in this case, is easier to create using scripts. Also the consumer gets to decides the format in which the value is used?